src/cli/serve/serve.js

```diff
@@ -124,17 +114,24 @@ function maybeAddConfig(name, configs, method) {
 function maybeSetRecentConfig(file, mode, configs, method) {
```

Comment:
I've changed this method's behaviour to rewrite `serverless.recent.yml` only to include the settings 

```YAML
xpack.serverless.plugin.developer.projectSwitcher.enabled: true
serverless: ${selectedProjectType}
```

@clintandrewhall, do you think this is a good approach?